src/apps/pillarx-app/components/TokensHorizontalTile/TokensHorizontalTile.tsx

```diff
@@ -69,9 +70,9 @@ if (!data || isDataLoading) {
     return (
         <div ref={divRef as React.RefObject<HTMLDivElement>}>
             <TileContainer className='flex-col px-10 pt-[30px] pb-5 tablet:p-5 mobile:p-0 mobile:bg-[#1F1D23]'>
-                <Body className='text-purple_light mb-2.5'>{tileTitle}</Body>
+                <Body className='text-purple_light mb-2.5'>{meta?.display.title}</Body>
```

Comment:
It's worth adding `?` after display also, there may be a that any of them are not defined

Reply:
done